Sasquatch/Sasquatch/Base.lproj/Main.storyboard

```diff
@@ -2008,7 +2008,7 @@
         <image name="distribute" width="23" height="23"/>
         <image name="fuel" width="23" height="23"/>
         <systemColor name="groupTableViewBackgroundColor">
-            <color red="0.94901960784313721" green="0.94901960784313721" blue="0.96862745098039216" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
```

Comment:
Why do we need to change systemColor?

Reply:
You are correct. These changes are unnecessary. I've removed them.